<ui version="4.0" >
 <class>Pref::Ui::InterfaceWidget</class>
 <widget class="QWidget" name="Pref::Ui::InterfaceWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>221</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>키보드</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>키보드를 이용한 단축키를 설정합니다.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="shortcut_tree" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>설명</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>단축키</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>221</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>마우스</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>마우스를 이용한 조작을 설정합니다.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>행동 설정</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>왼쪽 버튼 더블 클릭시</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="double_action_combo" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>가운데 버튼 클릭시</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="middle_action_combo" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>휠 스크롤시</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="scroll_action_combo" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>221</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>조작 단계</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>단계별로 조작하는 값을 설정합니다.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>앞으로/뒤로</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="seek_spin" >
           <property name="suffix" >
            <string> 초</string>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>300</number>
           </property>
           <property name="value" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>많이 앞으로/뒤로</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="seek_more_spin" >
           <property name="suffix" >
            <string> 초</string>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>600</number>
           </property>
           <property name="value" >
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>더 많이 앞으로/뒤로</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="seek_much_more_spin" >
           <property name="suffix" >
            <string> 초</string>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>600</number>
           </property>
           <property name="value" >
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>소리 크게/작게</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="volume_spin" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rsclist.qrc" />
 </resources>
 <connections/>
</ui>
